pkg/kubectl/cmd/patch.go

```diff
@@ -180,6 +180,7 @@ func RunPatch(f cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []strin
 		if !options.Local {
 			dataChangedMsg := "not patched"
 			helper := resource.NewHelper(client, mapping)
+			emptyPatchObj, err := helper.Patch(namespace, name, patchType, []byte("{}"))
```

Comment:
if a PATCH request was sent to the server and an object was returned that differed from the last known object kubectl has, I'm fine with a "patched" message being output. we don't want to do a compare like this.